metaci/testresults/tests/test_robot_importer.py

```diff
@@ -0,0 +1,34 @@
+from pathlib import PurePath
+
+import pytest
+from django.test import TestCase
+
+from metaci.conftest import BuildFlowFactory
+from metaci.testresults import models, robot_importer
+
+
+@pytest.mark.django_db
+class RobotImporterTestCase(TestCase):
+    def setUp(self):
+        pass
+
+    def test_nested_suites(self):
+        buildflow = BuildFlowFactory()
+        path = PurePath(__file__).parent / "robot_with_nested_suites.xml"
+        robot_importer.import_robot_test_results(buildflow, path)
+        assert models.TestResult.objects.all(), "Test results should have been created"
```

Comment:
Shouldn't this do more than just check that _some_ results were created?

Reply:
Yes I will add duration checks when I fix the bugs in the duration code. I can't really commit to deep checks of the overall correctness of the parsed data because I need to get back to performance work.